acceptance_tests/features/steps/inbox_internal.py

```diff
@@ -117,18 +116,10 @@ def internal_user_has_unread_message_in_inbox(context):
 
 @then('they are able to distinguish that the message is unread')
 def internal_user_can_distinguish_the_message_is_unread(_):
-    wait_for(_validate_message_unread_on_page, 5, 1)
+    wait_for(named_element_on_page, 8, 1, 'message-unread')
```

Comment:
This should read `wait_for(named_element_on_page, timeout=5, limit=1, argv='message-unread')`